content/6_Advanced/SPneg.mdx

```diff
@@ -118,7 +118,80 @@ subpart of SSSP Negative -->
 
 #### Solution
 
-<IncompleteSection />
+As mentioned in [cp-algorithms](https://cp-algorithms.com/graph/finding-negative-cycle-in-graph.html), we run Bellman-Ford $N$ times (instead of $N-1$ times). If we perform an update on the $N$th iteration, there is an negative cycle.
```

Comment:
Do you mean you **relax the edges** N times? "Run Bellman-Ford N times" kinda sounds like the algorithm is O(N^3). You can also say "do N iterations of the Bellman-Ford algorithm" like the cp-algorithms article does